### ISSUE TITLE
Missing sound for event: minecraft: on mac

### ISSUE THREAD

**Issue by Kokonico** [killpowa/Twickt-Launcher#1353]
**Describe the bug**
none of the new minecraft sounds for 1.19 are downloaded when creating a new 1.19 instance on mac

**To Reproduce**
1. Go to instances
2. Click on create new instance
3. create a new 1.19 instance and launch it.

**Expected behavior**
all sounds for wardens, allays, and goat horns should load.

**Possible solutions**
the only thing i know is that if i create a new instance in the vanilla launcher it works.

**Operating System:**
- OS: MacOS only
- Version: 1.1.23
- Java Version: all
- Minecraft Version: 1.19
- Mod Loader: all

**additional content**
full list of errors:
[17:50:57] [Worker-Main-11/WARN]: Found 'parent' loop while loading model 'minecraft:item/bow' in chain: minecraft:item/bow -> minecraft:item/bow
[17:51:00] [Worker-Main-8/ERROR]: Using missing texture, file minecraft:textures/item/crossbow.png not found
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.allay.ambient_with_item
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.allay.ambient_without_item
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.allay.death
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.allay.hurt
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.allay.item_given
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.allay.item_taken
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.allay.item_thrown
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:item.bucket.empty_tadpole
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:item.bucket.fill_tadpole
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.frog.tongue
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.horn_break
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.0
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.1
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.2
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.3
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.4
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.5
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.6
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.7
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:music_disc.5
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:music.overworld.deep_dark
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:music.overworld.swamp
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:music.overworld.jungle_and_forest
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:music.overworld.old_growth_taiga
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.parrot.imitate.warden
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.tadpole.grow_up
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.agitated
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.ambient
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.angry
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.attack_impact
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.death
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.dig
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.emerge
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.heartbeat
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.hurt
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.listening
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.listening_angry
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.nearby_close
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.nearby_closer
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.nearby_closest
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.roar
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.sniff
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.sonic_boom
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.sonic_charge
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.step
[17:51:04] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.tendril_clicks

**Comment by Eskaan** [comment]
Did you try to reinstall the instance (repair it)?

**Comment by Kokonico** [comment]
yes i have. i found a temporary fix by making a texture pack of the original files. making new instances or repairing old ones does not work.

**Comment by infonauten** [comment]
I have the exact same problem, also on Mac (similar setup to above): exact same warnings in my log file, no sounds playing for Allays / Warden / Goat Horns. Repairing the instance does not work.

**Comment by brandonkthomas** [comment]
Same issue for me on Windows, game version 1.19.2. All my instances of both Fabric and Vanilla flavors do not include assets for many of 1.19's new sounds (goat horn, some Warden sounds, Allay, "5" disc). 

Here's a full log list of the missing sounds:

```
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:entity.allay.ambient_with_item
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:entity.allay.ambient_without_item
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:entity.allay.death
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:entity.allay.hurt
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:entity.allay.item_given
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:entity.allay.item_taken
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:entity.allay.item_thrown
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:item.bucket.empty_tadpole
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:item.bucket.fill_tadpole
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:entity.frog.tongue
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.horn_break
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.0
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.1
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.2
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.3
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.4
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.5
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.6
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.sound.7
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:music_disc.5
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:music.overworld.deep_dark
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:music.overworld.swamp
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:music.overworld.jungle_and_forest
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:music.overworld.old_growth_taiga
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:entity.tadpole.grow_up
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.sonic_boom
[19:59:21] [Render thread/WARN]: Missing sound for event: minecraft:entity.warden.sonic_charge
```


@Eskaan I'm checking in 4 months later... was a fix ever discovered for this? The only thing I've seen thus far that works is manually downloading a 600MB ZIP of all the game's assets from https://mcasset.cloud and applying that as a resource pack.

"Repairing" instances does nothing, and creating new instances does nothing.

**Comment by Eskaan** [comment]
As far as I know, deleting the assets and then repairing fixes it.

**Comment by brandonkthomas** [comment]
> As far as I know, deleting the assets and then repairing fixes it.

Confirmed that this fixes the issue. I deleted the following folder: 

`C:\Users\username\AppData\Roaming\gdlauncher-next\datastore\assets`

The launcher auto-downloaded 1.19 assets on next modpack launch.

I assume this issue is due to the launcher initially downloading 1.18 assets when a 1.18 modpack is launched, then never updating said assets (or partially updating them) when a 1.19 modpack is launched.
